gcli/tests/common/mod.rs

```diff
@@ -70,16 +74,31 @@ pub async fn create_messager() -> Result<Node> {
     node.wait(logs::gear_node::IMPORTING_BLOCKS)?;
 
     let messager = env::wasm_bin("messager.opt.wasm");
-    let _ = gear(&[
+
+    let args = &[
         "-e",
         &node.ws(),
         "upload-program",
         &messager,
-        "0x",
-        "0x",
-        "0",
-        "10000000000",
-    ])?;
+        "--gas-limit",
+        "20000000000",
+    ];
+
+    let _ = gear(
+        #[cfg(feature = "vara-testing")]
+        args,
+        #[cfg(not(feature = "vara-testing"))]
+        &[
+            args.to_vec(),
+            vec![
```

Comment:
First of all I'd separate args creation above by feature like that:
```rust
#[cfg(feature = "vara-testing")]
let args = ...

#[cfg(not(feature = "vara-testing"))]
let args = ...
```
Apply similar approach across all tests.
Furthermore, I'd use some constructor for such vectors of args to increase readability (or at least format macro)